frontend/src/metabase/admin/settings/components/widgets/EmbeddingOption/EmbeddingOption.module.css

```diff
@@ -0,0 +1,9 @@
+.icon {
```

Comment:
Fix icons not following custom brand color (EE)

<img src="https://github-production-user-asset-6210df.s3.amazonaws.com/1937582/360983532-797cde34-fa5b-4e3b-8799-596c707e1411.png?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAVCODYLSA53PQK4ZA%2F20240830%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20240830T162656Z&X-Amz-Expires=300&X-Amz-Signature=af17e8d87c83dc9d857af8649dd09721ed92f1029b0d3a10bf4c1940383647b0&X-Amz-SignedHeaders=host&actor_id=1937582&key_id=0&repo_id=30203935" />